bzt/modules/blazemeter.py

```diff
@@ -72,6 +75,8 @@ def prepare(self):
         self.client.timeout = dehumanize_time(self.settings.get("timeout", self.client.timeout))
         self.send_interval = dehumanize_time(self.settings.get("send-interval", self.send_interval))
         self.send_monitoring = self.settings.get("send-monitoring", self.send_monitoring)
+        self.monitoring_buffer_limit = self.settings.get("monitoring-buffer-limit",
```

Comment:
Usual style is to put original field value as default, especially when you set it to constant already